### PR TITLE
Remove 'ol' old content -- not used by this site

### PR DESCRIPTION
**Pull request by masinter**

### CODE REVIEW

File: content/en/project/stories.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: Our Stories
 weight: 40
 type: docs
 ---
+Many of these weren from posts on one the Interlisp mailing lists, and others written expressedly. What are **you** using Medley for?
```
Comment (nbriggs) [inline content/en/project/stories.md]: This sentence doesn't read correctly.

Comment (masinter) [reply content/en/project/stories.md]: leaving stores alone. If Ron started a sentence with And, shame on him.

Comment (nbriggs) [reply content/en/project/stories.md]: Not an "And..." issue: "weren" should be "were", and "expressedly" (not a word) should be "expressly"

Comment (nbriggs) [reply content/en/project/stories.md]: Well, "expressedly" is an archaic form, as in:

>Adverb. expressedly (comparative more expressedly, superlative most expressedly) (chiefly nonstandard or archaic) In a way that is expressed or made explicit.

although it is usually a mispelling of expressly.

File: content/en/software/install-and-run/running-on-linux.md [hunk 2]
```diff
@@ -19,10 +19,10 @@ Medley can be installed on your Linux system in one of two configurations: *stan
 
 ## **Standard Installation \(Debian-based systems only\)**
 
-We are currently only building standard installations for Debian-based systems (i.e., systems that support dpkg).
+We are currently only building standard installations for Debian-based systems (i.e., systems that support dpkg). If you need something else, let us know.
```
Comment (stumbo) [inline content/en/software/install-and-run/running-on-linux.md]: Should this be submit an issue?

File: content/en/software/install-and-run/running-on-linux.md [hunk 3]
```diff
@@ -19,10 +19,10 @@ Medley can be installed on your Linux system in one of two configurations: *stan
...
 
-In a standard installation, Medley is installed in system directories
-(specifically, /usr/local/interlisp). Support-like man pages and (a link to) the medley executable are also installed in standard system locations (e.g., /usr/local/man and /usr/local/bin).
+In a standard Linux installation, Medley is installed in system directories
+under /usr/local/interlisp). Man pages and (links to) the medley executable are also installed in standard system locations (e.g., /usr/local/man and /usr/local/bin).
```
Comment (stumbo) [inline content/en/software/install-and-run/running-on-linux.md]: Remove the ending ) after interlisp.

File: content/en/software/install-and-run/running-on-linux.md [hunk 4]
```diff
@@ -119,7 +119,7 @@ Also note that with local installations, `man medley` will not work.  However, a
 	ubuntu@oio:~$ tar -C <interlisp_directory> -xzf <tar_filepath>
 	```
 
-4. Run Medley
+4. [ ] Run Medley
```
Comment (stumbo) [inline content/en/software/install-and-run/running-on-linux.md]: Are the square brackets here on purpose?  They give you a checkbox.

**Review by stumbo — CHANGES_REQUESTED** [review]
A couple minor things...

File: content/en/software/install-and-run/running-on-linux.md (hunk 2 above)
Comment (nbriggs) [reply content/en/software/install-and-run/running-on-linux.md]: I agree.  Can we provide a clean link to the issue submission github page?

File: content/en/software/install-and-run/running-on-linux.md (hunk 3 above)
Comment (masinter) [reply content/en/software/install-and-run/running-on-linux.md]: thx got it

File: content/en/software/install-and-run/running-on-linux.md (hunk 2 above)
Comment (masinter) [reply content/en/software/install-and-run/running-on-linux.md]: i took back out the offending sentence. In general  we're not offering support or any particular enhancements, or even promising we'll accept things that don't meet our standards.

File: content/en/software/install-and-run/running-on-linux.md (hunk 4 above)
Comment (masinter) [reply content/en/software/install-and-run/running-on-linux.md]: removed

**Review by stumbo — APPROVED** [review]

**Pull request merged**